modules/vaos/app/models/vaos/appointment_time_slot.rb

```diff
@@ -0,0 +1,24 @@
+# frozen_string_literal: true
+
+require 'common/models/resource'
+
+module VAOS
+  class AppointmentTimeSlot < Common::Resource
+    transform_types do |type|
+      case type.name
+      when :start_date_time || :end_date_time
+        type.constructor do |value|
+          DateTime.strptime(value, '%m/%d/%Y %H:%M:%S')
+        end
+      else
+        type
+      end
+    end
```

Comment:
Please take a look at the AppointmentTime < Virtus::Attributes. Almost every VAMF case is probably captured there. I think this ends up being code duplication by introducing dry-types prematurely.